mixins/arrow-keys-mixin.md

```diff
@@ -51,4 +51,8 @@ customElements.define('my-element', MyElement);
 
 - `arrowKeysDirection` (String): Indicates which arrow keys are allowed (default is leftright)
 - `arrowKeysNoWrap` (Boolean): Whether focus should wrap from end-to-start and start-to-end
-- `arrowKeysBeforeFocus` (Function): Async callback invoked before focus us applied
+
+**Functions:**
+- `arrowKeysContainer`: Hooks up the event listener, to be used in your `render` function to wrap the focusable elements
```

Comment:
I think we usually include the argument when we list the methods. Ex. [lists](https://github.com/BrightspaceUI/core/tree/main/components/list#methods).  Not sure if we're entirely consistent though. So I think I would write this as "`arrowKeysContainer(TemplateResult)`: ...." or "`arrowKeysContainer(htmlTemplate)`: ....", or something along those lines.

Reply:
Took another stab at this section!  Yeah we don't seem to be super consistent, tried to copy the general style I saw on a few pages.  Feel free to suggest edits, not tied to any of them lol